components/dashboard/src/experiments.ts

```diff
@@ -28,6 +28,7 @@ const Experiments = {
      * Experiment "example" will be activate on login for 10% of all clients.
      */
     // "example": 0.1,
+    "login-from-context-6826": 0.5,
```

Comment:
Some documentation incl. a link to this PR would be nice!

Reply:
Also, side question: Can you manually "force" A or B to test code-dev deployments?

Reply:
> Can you manually "force" A or B to test code-dev deployments?

Sure, you have to manipulate `localStorage` in your web dev tools. They are configured to be stable atm, so will happily accept your changes.

**Update**: just recognized that this is not true, actually! And found a bug in the impl. Will fix after lunch. :upside_down_face:

Reply:
PR: https://github.com/gitpod-io/gitpod/pull/7103